src/async_fuse/memfs/kv_engine/etcd_impl.rs

```diff
@@ -207,11 +434,117 @@ impl KVEngine for EtcdKVEngine {
         }
     }
 
+    /// Range get, return all key-value pairs start with prefix
     async fn range(&self, prefix: &KeyType) -> DatenLordResult<Vec<ValueType>> {
         let _timer = KV_METRICS.start_kv_operation_timer("range");
         let result = self.range_raw_key(prefix.to_string_key()).await?;
         Ok(result)
     }
+
+    /// Revokes a lease
+    async fn lease_revoke(&self, lease_id: i64) -> DatenLordResult<()> {
```

Comment:
expose revoke might not be neccessary,  since we only need to drop Session ?

Reply:
Ok, that make sense.